XmlFormatter/src/Manager/VersionManager.cs

```diff
@@ -81,13 +80,17 @@ public async Task<VersionCompare> GitHubVersionIsNewer()
             {
                 ThrowError("No version found", "Could't not find a version on GitHub");
             }
-            Release latestRelease = releases[0];
-            string latest = latestRelease.Name;
-            Regex regex = new Regex(@"([0-9]{1,}.[0-9]{1,}.[0-9]{1,})");
+            string gitHubString = "";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'gitHubString'.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5553158)